src/FlutureTMonetEither/index.js

```diff
@@ -366,7 +366,14 @@ FlutureTMonetEither.prototype.fork = function fork(leftFn, rightFn) {
  */
 FlutureTMonetEither.prototype.and = function and(futureEither) {
   return this.constructor.of(
-    this.run.and(futureEither.run)
+    Future.of(curry((e1, e2) => {
```

Comment:
LGTM
The only optimization in this case is that I would use native currying here to remove surrogate `curry` overhead
```javascript
Future.of(either1 => either2 => {
...
});